### PR TITLE
Mount point asterisk

### PR DESCRIPTION
**Pull request by jreidinger**
how it looks:

snimek obrazovky_2018-09-03_19-16-40
snimek obrazovky_2018-09-03_19-16-18

### CODE REVIEW

File: src/lib/y2partitioner/widgets/blk_devices_table.rb [hunk 1]
```diff
@@ -200,7 +202,12 @@ def filesystem_label_value(device)
 
       def mount_point_value(device)
         fs = filesystem(device)
-        fs.nil? ? "" : fs.mount_path
+        res = fs.nil? ? "" : fs.mount_path
+        if fs && Yast::Mode.normal
+          res += "(*)" if fs.mount_point && !fs.mount_point.active?
+        end
+
+        res
```
[the remark this answers is not in the document]
Comment (jreidinger) [reply src/lib/y2partitioner/widgets/blk_devices_table.rb]: fine for me with changing it to " *"

File: src/lib/y2partitioner/widgets/blk_device_attributes.rb [hunk 2]
```diff
@@ -168,7 +169,14 @@ def device_filesystem
       # @return [String]
       def device_filesystem_mount_point
         # TRANSLATORS: Mount point information, where %s is replaced by a mount point
-        format(_("Mount Point: %s"), blk_device.filesystem_mountpoint || "")
+        res = format(_("Mount Point: %s"), blk_device.filesystem_mountpoint || "")
+        return res unless Yast::Mode.normal
```
[the remark this answers is not in the document]
Comment (joseivanlopez) [reply src/lib/y2partitioner/widgets/blk_device_attributes.rb]: But during installation, all new mount points are always active, so the asterisk will be not shown in that case.

File: src/lib/y2partitioner/widgets/blk_device_attributes.rb [hunk 3]
```diff
@@ -168,7 +169,14 @@ def device_filesystem
...
+        return res unless blk_device.blk_filesystem
+        return res unless blk_device.blk_filesystem.mount_point
+        # TRANSLATORS: note appended to mount point if mount point is not now mounted
+        res += _(" (not mounted)") unless blk_device.blk_filesystem.mount_point.active?
+
+        res
```
[the remark this answers is not in the document]
Comment (joseivanlopez) [reply src/lib/y2partitioner/widgets/blk_device_attributes.rb]: This version is difficult to read for me. What about adding a new auxiliary method that checks whether it is necessary to add the ending note?

File: src/lib/y2partitioner/widgets/blk_device_attributes.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (jreidinger) [reply src/lib/y2partitioner/widgets/blk_device_attributes.rb]: I think only when they are in staging, so situation that they are not in staging cannot happen?

**Pull request merged**